src/patterns/download-resources/index.njk

```diff
@@ -0,0 +1,13 @@
+---
+title: Download resources
+group: Help users to...
+experimental: true
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+Allow users to filter and sort downloadable resources.
+
+## Example
+{{
+    patternlibExample({ "path": "patterns/download-resources/examples/download-resources/index.njk" })
+}}
```

Comment:
"Research on this component" section needs to be added to fit with the style of other patterns and components. I asked you to take out the panel import because it wasn't being used but you'll need to bring it back in for this